server/xpub/entities/manuscript/resolvers.js

```
@@ -237,6 +237,22 @@ ${err}`,
       return team.teamMembers[0].alias
     },
 
+    async clientStatus(manuscript) {
+      let clientStatus = 'REJECTED'
```

Probably shouldn't default to `REJECTED`. Maybe throw an error instead?